core-bundle/tests/EventListener/PrettyErrorScreenListenerTest.php

```diff
@@ -306,14 +295,20 @@ public function testDoesNotLogUnloggableExceptions(): void
     /**
      * @param Environment&MockObject $twig
      */
-    private function getListener(string $userClass, bool $expectLogging = false, Environment $twig = null): PrettyErrorScreenListener
+    private function getListener(bool $isBackendUser, bool $expectLogging = false, Environment $twig = null): PrettyErrorScreenListener
```

Comment:
We could use `$isBackendUser = false` and call `$this->getListener()` instead of `$this->getListener(false)` above.

Reply:
You mean a default value?

Reply:
Yes.

Reply:
will do

Reply:
Changed in 919f9744ba256494623213ff449d21c1b4b78143